kubernetes/src/kubernetes/config/incluster_config.rb

```diff
@@ -36,6 +36,10 @@ def validate
       raise ConfigError.new("Service token file does not exists") unless File.file?(self.ca_cert)
     end
 
+    def self.is_in_cluster()
```

Comment:
This should be `def in_cluster?` its pretty un-ruby like to use `is`  methods.

Reply:
done.